### DISCUSSION TITLE
我提供了一种更为快捷的到达该网站的方式

### DISCUSSION THREAD

**Discussion by NatsukoYamamura** [share121/inter-knot#1269] [其他]
直接访问
[interknot.work](https://interknot.work)
即可到达

![image](https://github.com/user-attachments/assets/df1eb579-69a9-4201-8070-7453f5872c31)

碎碎念：
因为存在于github.io下的网站未备案，所以无法通过aliyun的重定向URL，故又花费199大洋购买了一台（其实早就买了233）的云服务器作为重定向服务器。。。

顺带宣传一下，我自己的个人站 [yukihasu.work](http://yukihasu.work) 
祝大家在绳网玩得开心

**Comment by AetherLude** [comment]
好！

**Reply by NatsukoYamamura** [reply]
谢谢

**Comment by Skylerluo** [comment]
厉害👍🏻

**Reply by NatsukoYamamura** [reply]
谢谢

**Comment by su2569** [comment]
是重定向(悲

**Reply by NatsukoYamamura** [reply]
没办法，本人没有能力去真正建立并维护一个基于自己云服务器的论坛，up给的这个基于git的已经较好用了

**Comment by shenguojia123** [comment]
你好，我安装了绳网跨域助手但还是打不开绳网怎么办
![image](https://github.com/user-attachments/assets/9238f1e6-d806-4f60-9e3b-f788d52be90b)
![image](https://github.com/user-attachments/assets/357cc0f7-efe2-4d33-872c-7e7272893468)

**Reply by NatsukoYamamura** [reply]
请更新绳网小助手（最新1.3.0）并确认您开启了油猴在该网站的使用权限
![image](https://github.com/user-attachments/assets/35c59e95-9a9b-4674-a800-344b3859c548)

**Reply by shenguojia123** [reply]
好的，谢谢

**Comment by talataos** [comment]
你和法厄同什么关系

**Reply by NatsukoYamamura** [reply]
我只是一个路人，今天刚注册绳网账号，就看到了这么多热心的群众，太感动了

**Comment by xiaochenyu24** [comment]
太完美了

**Reply by NatsukoYamamura** [reply]
谢谢

**Comment by VacuolePaoo** [comment]
感谢绳匠[**sunnycherry2023**](https://github.com/sunnycherry2023)做出的贡献！
我们已经支持使用新的域名 [绳网.top](https://绳网.top) 访问，小助手也会发布新的版本。
原GithubPage地址可用，interknot.work仍可重定向

**Reply by NatsukoYamamura** [reply]
话说这是直接上云了嘛？看归属地是 中国–山东–临沂 移动的

**Comment by hzjwork** [comment]
大佬牛波一

**Comment by xxtg666** [comment]
其实没必要买服务器，用cloudflare就可以，还免费（

**Comment by dengyunzhe** [comment]
厉害了
